src/App.css

```diff
@@ -1,10 +1,188 @@
+@keyframes flicker {
+
+  0%,
+  18%,
+  22%,
+  25%,
+  53%,
+  57%,
+  100% {
+    text-shadow:
+      0 0 2px #fff,
+      0 0 4px #fff,
+      0 0 8px #fff,
+      0 0 15px #0fa,
+      0 0 30px #0fa,
+      0 0 40px #0fa,
+      0 0 70px #0fa,
+      0 0 100px #0fa;
+  }
+
+  20%,
+  24%,
+  55% {
+    text-shadow:
+      0 0 1px #fff,
+      0 0 2px #fff,
+      0 0 5px #fff,
+      0 0 11px #0fa,
+      0 0 20px #0fa,
+      0 0 30px #0fa,
+      0 0 55px #0fa,
+      0 0 80px #0fa;
+  }
+}
+
 body {
   margin: 0;
   overflow: hidden;
-  background: rgb(17,17,19);
+  /* background: rgb(17,17,19); */
```

Comment:
So bad! You forgot to remove a commented line!

Reply:
I'm really sorry. I didn't check CSS for comments!
I will be more attentive.

Reply:
done